src/components/Photo.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+function Photo({ url }){
+  return (
+    <img src={url} /> 
+  );
+
+}
+
+Photo.propTypes = {
+  photo:PropTypes.array.isRequired
```

Comment:
```suggestion
  url: PropTypes.string.isRequired
```
Here the prop that you want to validate is url which was passed from the parent Photos component.